config/locales/es.yml

```diff
@@ -298,6 +298,39 @@ es:
       start: Empezar
       welcome_to_daria: "¡Bienvenido a DARIA (edición de %{fund})!"
       what_line: "¿En cuál línea estás trabajando?"
+  multi_factor:
```

Comment:
Go ahead and put the English strings in for now and we'll tee up a fluent Spanish speaker to put in proper translations!

Reply:
Done